dev/wlp-jakartaee-transform/rules/jakarta-xml-dd.properties

```diff
@@ -31,20 +31,39 @@ javax.ws.rs.Application=jakarta.ws.rs.Application
 javax.validation.constraints.Min=jakarta.validation.constraints.Min
 
 #Faces Configurations
-javax.faces.webapp.FacesServlet=jakarta.faces.webapp.FacesServlet
-javax.faces.PROJECT_STAGE=jakarta.faces.PROJECT_STAGE
-javax.faces.ENABLE_WEBSOCKET_ENDPOINT=jakarta.faces.ENABLE_WEBSOCKET_ENDPOINT
 javax.faces.ALWAYS_PERFORM_VALIDATION_WHEN_REQUIRED_IS_TRUE=jakarta.faces.ALWAYS_PERFORM_VALIDATION_WHEN_REQUIRED_IS_TRUE
-javax.faces.FACELETS_LIBRARIES=jakarta.faces.FACELETS_LIBRARIES
-javax.faces.WEBAPP_RESOURCES_DIRECTORY=jakarta.faces.WEBAPP_RESOURCES_DIRECTORY
-javax.faces.DISABLE_FACESSERVLET_TO_XHTML=jakarta.faces.DISABLE_FACESSERVLET_TO_XHTML
-javax.faces.VIEWROOT_PHASE_LISTENER_QUEUES_EXCEPTIONS=jakarta.faces.VIEWROOT_PHASE_LISTENER_QUEUES_EXCEPTIONS
+javax.faces.AUTOMATIC_EXTENSIONLESS_MAPPING=jakarta.faces.AUTOMATIC_EXTENSIONLESS_MAPPING
+javax.faces.CLIENT_WINDOW_MODE=jakarta.faces.CLIENT_WINDOW_MODE
 javax.faces.CONFIG_FILES=jakarta.faces.CONFIG_FILES
-javax.faces.STATE_SAVING_METHOD=jakarta.faces.STATE_SAVING_METHOD
-javax.faces.validator.ENABLE_VALIDATE_WHOLE_BEAN=jakarta.faces.validator.ENABLE_VALIDATE_WHOLE_BEAN
+javax.faces.DATETIMECONVERTER_DEFAULT_TIMEZONE_IS_SYSTEM_TIMEZONE=jakarta.faces.DATETIMECONVERTER_DEFAULT_TIMEZONE_IS_SYSTEM_TIMEZONE
+javax.faces.DEFAULT_SUFFIX=jakarta.faces.DEFAULT_SUFFIX
+javax.faces.DISABLE_FACESSERVLET_TO_XHTML=jakarta.faces.DISABLE_FACESSERVLET_TO_XHTML
+javax.faces.ENABLE_WEBSOCKET_ENDPOINT=jakarta.faces.ENABLE_WEBSOCKET_ENDPOINT
+javax.faces.FACELETS_BUFFER_SIZE=jakarta.faces.FACELETS_BUFFER_SIZE
+javax.faces.FACELETS_DECORATORS=jakarta.faces.FACELETS_DECORATORS
+javax.faces.FACELETS_LIBRARIES=jakarta.faces.FACELETS_LIBRARIES
+javax.faces.FACELETS_REFRESH_PERIOD=jakarta.faces.FACELETS_REFRESH_PERIOD
+javax.faces.FACELETS_RESOURCE_RESOLVER=jakarta.faces.FACELETS_RESOURCE_RESOLVER
+javax.faces.FACELETS_SKIP_COMMENTS=jakarta.faces.FACELETS_SKIP_COMMENTS
+javax.faces.FACELETS_SUFFIX=jakarta.faces.FACELETS_SUFFIX
+javax.faces.FACELETS_VIEW_MAPPINGS=jakarta.faces.FACELETS_VIEW_MAPPINGS
+javax.faces.FULL_STATE_SAVING_VIEW_IDS=jakarta.faces.FULL_STATE_SAVING_VIEW_IDS
+javax.faces.HONOR_CURRENT_COMPONENT_ATTRIBUTES=jakarta.faces.HONOR_CURRENT_COMPONENT_ATTRIBUTES
 javax.faces.INTERPRET_EMPTY_STRING_SUBMITTED_VALUES_AS_NULL=jakarta.faces.INTERPRET_EMPTY_STRING_SUBMITTED_VALUES_AS_NULL
-javax.faces.WEBAPP_CONTRACTS_DIRECTORY=jakarta.faces.WEBAPP_CONTRACTS_DIRECTORY
-javax.faces.CLIENT_WINDOW_MODE=jakarta.faces.CLIENT_WINDOW_MODE
+javax.faces.LIFECYCLE_ID=jakarta.faces.LIFECYCLE_ID
+javax.faces.NUMBER_OF_CLIENT_WINDOWS=jakarta.faces.NUMBER_OF_CLIENT_WINDOWS
```

Comment:
Added in 4.0. Remove `javax.faces.NUMBER_OF_CLIENT_WINDOWS=jakarta.faces.NUMBER_OF_CLIENT_WINDOWS`